lib/bulk_data_test_kit/v1.0.1/bulk_data_export_kick_off_test.rb

```diff
@@ -34,6 +34,17 @@ def self.properties
         polling_url = export_kick_off_success
         output polling_url:
       end
+
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@28',
+                            'hl7.fhir.uv.bulkdata_2.0.0@32',
```

Comment:
this test is calling [export_kick_off_success](https://github.com/inferno-framework/bulk-data-test-kit/blob/2f54b3ee569db62b1b4650a7479c14640e9262ea/lib/bulk_data_test_kit/v1.0.1/bulk_data_export_kick_off_test.rb#L34C42-L34C43) which calls [perform_export_kick_off_request](https://github.com/inferno-framework/bulk-data-test-kit/blob/2f54b3ee569db62b1b4650a7479c14640e9262ea/lib/bulk_data_test_kit/export_operation_tests.rb#L86). In that function it appears to take the parameters from a hash and then concatenate them into a single parameter in the URL. If this is not correct I could move this elsewhere or remove it

Reply:
thank you for pointing me there!

Here's req 31:
> A client MAY repeat kick-off parameters that accept comma delimited values multiple times in a kick-off request.

Here's req 32:
> The server SHALL treat the values provided as if they were comma delimited values within a single instance of the parameter.

From what I understand, this requirement is saying:

> Assume there's a parameter that takes multiple comma-delimited values: `_type=Patient,Observation`
> 
> If the client provides the url `www.example.com?_type=Patient&_type=Observation` (repeating the parameter with a different value)
> Then the server SHALL interpret this as `www.example.com?_type=Patient,Observation`

Because this is getting the parameters from a hash, the parameter names cannot be repeated in this request to the server, so the test is not capable of testing this requirement.

Please remove this requirement.

Thanks!